lib/puppet/type/java_ks.rb

```diff
@@ -59,50 +58,48 @@ def insync?(is)
     end
 
     newparam(:target) do
-      desc 'Destination file for the keystore.  We autorequire the parent
-        directory for convenience.'
+      desc 'Destination file for the keystore.  This is an autorequire.'
 
       isnamevar
     end
 
     newparam(:certificate) do
-      desc 'An already signed certificate that we can place in the keystore.  We
-        autorequire the file for convenience.'
+      desc 'An already signed certificate that we can place in the keystore.  This is an autorequire.'
```

Comment:
Parameters aren't autorequires, they autorequire something, and the docs should specify what is being autorequired.